net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/bgp.volt

```diff
@@ -1 +1,103 @@
-{{ partial("layout_partials/base_form",['fields':generalForm,'id':'frm_ospf_settings'])}}
+{#
+
+OPNsense® is Copyright © 2014 – 2017 by Deciso B.V.
+This file is Copyright © 2017 by Fabian Franz
```

Comment:
I copied your file and replaced ospf with bgp, it's yours :)
I added a noted with my name